src/main/java/se/sundsvall/emailreader/integration/db/entity/AttachmentEntity.java

```diff
@@ -11,6 +11,8 @@
 import jakarta.persistence.PrePersist;
 import jakarta.persistence.Table;
 
+import org.hibernate.Length;
+
 import se.sundsvall.dept44.util.jacoco.ExcludeFromJacocoGeneratedCoverageReport;
 
```

Comment:
` @ExcludeFromJacocoGeneratedCoverageReport` could be removed from prePersist(). This is very easy to test.

```
entity.prePersist();
assertThat(entity.getCreatedAt()).isCloseTo(now(), within(1, SECONDS));
```